icu4c/source/test/cintltst/cloctst.c

```diff
@@ -6018,6 +6019,47 @@ static void TestToLanguageTag(void) {
     }
 }
 
+static void TestBug20132(void) {
+    char langtag[256];
+    UErrorCode status;
+    int32_t len;
+
+    static const char inloc[] = "C";
+    static const char expected[] = "en-US-u-va-posix";
+    const int32_t expected_len = uprv_strlen(expected);
+
+    /* Before ICU-20132 was fixed, calling uloc_toLanguageTag() with a too small
```

Comment:
FYI  You can use // comments in C.

Reply:
Ah, C99, I had totally forgotten about that. 😉